nautobot/utilities/utils.py

```diff
@@ -844,6 +844,13 @@ def get_all_lookup_expr_for_field(model, field_name):
                     "name": build_lookup_label(name, field.lookup_expr),
                 }
             )
+        elif name == field_name and not name.startswith("has_"):
```

Comment:
This is to fix #3345? Can we fix it at its source rather than continuing to add to this function?